aten/src/ATen/native/cuda/int4mm.cu

```diff
@@ -1290,7 +1290,6 @@ at::Tensor _convert_weight_to_int4pack_cuda(
 
   TORCH_CHECK(in.dim() == 2);
   TORCH_CHECK(in.dtype() == at::kByte);
-  TORCH_CHECK(in.is_contiguous());
```

Comment:
Why are you removing this check? I.e. it should only work for contiguous  tensors, shouldn't it?

Reply:
`python test/test_meta.py -k test_dispatch_symbolic_meta_outplace_all_strides__convert_weight_to_int4pack_` cannot pass this check. Can I remove this check and use `auto weight = in.contiguous()` instead?

Reply:
May be right behavior would be to add `xfail` for all_strides tests, as this op is perf optimized and as such does not make much sense to use with non-contiguous tensors.

Reply:
Could you please tell me where to add `xfail` for all_strides tests?

Reply:
@malfet Could you please tell me where to add xfail for all_strides tests?